initfiles/componentfiles/configxml/esp.xsd.in

```diff
@@ -440,6 +440,15 @@
 	                </xs:appinfo>
 	              </xs:annotation>
 	            </xs:attribute>
+	            <xs:attribute name="invalidURLsAfterAuth" type="xs:string" use="optional" default="/esp/login">
+	              <xs:annotation>
+	                <xs:appinfo>
+	                  <width>50</width>
+	                  <tooltip>After authenticated, those URL requests will be redirected to home page.</tooltip>
```

Comment:
This tooltip is not clear. Let's work with Jim to reword

Reply:
After authentication, requests are redirected to the home page.